src/fenics_constitutive/stress_strain.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+from .interfaces import Constraint
+import ufl
+
+__all__ = ["ufl_mandel_strain", "strain_from_grad_u"]
+
+
+def ufl_mandel_strain(
+    u: ufl.core.expr.Expr, constraint: Constraint
+) -> ufl.core.expr.Expr:
+    """
+    Compute the Mandel-strain from the displacement field.
+
+    Parameters:
+        u: Displacement field.
+        constraint: Constraint that the model is implemented for.
+
+    Returns:
+        Vector-valued UFL expression of the mandel strain.
```

Comment:
Just wondering which documentation style you are following here?
We should be able to refer to some source and keep it consistent throughout the repo.

Minor suggestion following [this guide](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)
```suggestion
    Computes the Mandel-strain from the displacement field.

    Args:
        u: Displacement field.
        constraint: Constraint that the model is implemented for.

    Returns:
        Vector-valued UFL expression of the mandel strain.
```

Reply:
Oh, sorry about that. I haven't written a good docstring in a while. I would also suggest that we do the google convention for docstrings. Will fix this